llvm/lib/CodeGen/SelectionDAG/SelectionDAGISel.cpp

```diff
@@ -698,7 +698,7 @@ bool SelectionDAGISel::runOnMachineFunction(MachineFunction &mf) {
         Def->getParent()->insert(std::next(InsertPos), MI);
       } else
         LLVM_DEBUG(dbgs() << "Dropping debug info for dead vreg"
-                          << Register::virtReg2Index(Reg) << "\n");
+                          << Reg.virtRegIndex() << "\n");
```

Comment:
```suggestion
                          << printReg(Reg) << '\n');
```

Reply:
These technically make it not NFC since printReg includes the %.

Reply:
I wouldn't consider exact debug printing to be F. It should be included for debug log consistency anyway